Project.toml

```
@@ -0,0 +1,19 @@
+name = "UnitfulIntegration"
+uuid = "d000b014-1cca-4823-a42a-b615c320b663"
+authors = ["contributors"]
+version = "0.2.0"
+
+[deps]
+QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
+Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
+
+[compat]
+julia = "1"
+QuadGK = "2.6"
+Unitful = "^1.12"
```

```suggestion
Unitful = "1.12"
```
The caret is unnecessary.